engine.py

```diff
@@ -531,8 +537,8 @@ def _create_dialog(self, title, bundle, widget, parent):
             previous_parent = dialog.parent()
             try:
                 self.log_debug("Attempting to attach dialog to 3ds Max...")
-                # widget must be parentless when calling MaxPlus.AttachQWidgetToMax
                 dialog.setParent(None)
+                # widget must be parentless when calling MaxPlus.AttachQWidgetToMax
                 MaxPlus.AttachQWidgetToMax(dialog)
```

Comment:
Isn't this going to raise NameError if MaxPlus failed to import?

Reply:
Added this comment in the code:

```
# Accessing MaxPlus here is safe because we're inside a
# a branch of code that can only be executed on Max 2019 and lower.
```